src/core/Cargo.toml

```diff
@@ -44,7 +44,7 @@ md5 = "0.7.0"
 memmap2 = "0.9.4"
 murmurhash3 = "0.0.5"
 needletail = { version = "0.5.1", default-features = false }
-niffler = { version = "2.6.0", default-features = false, features = [ "gz" ] }
+niffler = { version = "2.4.0", default-features = false, features = [ "gz" ] }
```

Comment:
did you need to revert this for the plugin?

Reply:
yep

Reply:
(I figured no harm done since it will just come up again with dependabot ;))

Reply:
but I will document next time I stumble across the error.